### ISSUE TITLE
Create kong storage adapter for mysql

### ISSUE THREAD

**Issue by gr4per** [OpusCapitaBusinessNetwork/api-gw#1]
I guess kong would benefit from that so we could decide to spend the effort to do this for the kong community.
I expect changes compared to postgre should be minimal but for us it means one db cluster less to manage as we could use our mysql cluster then.

**Comment by patrykkopycinski** [comment]
https://github.com/Mashape/kong/compare/master...li-wl:master

**Comment by sureshvarman-cronj** [comment]
@patrykkopycinski the above link, doesnt seems like promising, because we end up completely dependency on https://github.com/li-wl/kong (external, not directly on kong).

**Comment by sureshvarman-cronj** [comment]
Yes, forked from kong repo, adding feature for custom adapter for any database, mysql adapter as a separate repo.
